.mega-linter.yml

```diff
@@ -1,6 +1,7 @@
 # analyze any source files except for packages we did not write ourselves
 FILTER_REGEX_INCLUDE: .*src/.*
-FILTER_REGEX_EXCLUDE: .*src/navigation/(pure_pursuit|particle_filter)(/.*)?
+FILTER_REGEX_EXCLUDE: .*src/control/pure_pursuit(/.*)?
```

Comment:
I think these need to all be in just one FILTER_REGEX_EXCLUDE, so you need to figure out the right regex for combining paths lol. OR, just reformat the files that are failing in CI and remove this flag all together